change/@fluentui-react-link-a0a06b2e-a83e-44b3-b2bd-f432395554c8.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
## [🕵 fluentuiv9 Open the Visual Regressions report to inspect the affected screenshots]( https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-approval/index.html?organization=uifabric&project=fabricpublic&prId=29336&commitId=3fadbbb91efd1f1f32a9e815a94d2cc012345813&env=prod&clientType=FLUENTUI)


  <details><summary><b>Link Converged - Rendered as anchor</b> 16 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Link Converged - Rendered as anchor.Inline - RTL.pressed.chromium.png|190| Changed|
|Link Converged - Rendered as anchor.Inline.pressed.chromium.png|190| Changed|
|Link Converged - Rendered as anchor.Inverted - High Contrast.pressed.chromium.png|451| Changed|
|Link Converged - Rendered as anchor.Inline Disabled Focusable.pressed.chromium.png|623| Changed|
|Link Converged - Rendered as anchor.Inverted disabled - High Contrast.pressed.chromium.png|564| Changed|
|Link Converged - Rendered as anchor.Inverted disabled - Dark Mode.pressed.chromium.png|909| Changed|
|Link Converged - Rendered as anchor.Inverted - Dark Mode.pressed.chromium.png|724| Changed|
|Link Converged - Rendered as anchor.Inverted disabled.pressed.chromium.png|516| Changed|
|Link Converged - Rendered as anchor.Inverted.pressed.chromium.png|411| Changed|
|Link Converged - Rendered as anchor.Stand-alone - Dark Mode.pressed.chromium.png|94| Changed|
|Link Converged - Rendered as anchor.Stand-alone - High Contrast.pressed.chromium.png|84| Changed|
|Link Converged - Rendered as anchor.Stand-alone - RTL.pressed.chromium.png|440| Changed|
|Link Converged - Rendered as anchor.Stand-alone Disabled Focusable - High Contrast.pressed.chromium.png|533| Changed|
|Link Converged - Rendered as anchor.Stand-alone.pressed.chromium.png|440| Changed|
|Link Converged - Rendered as anchor.Stand-alone Disabled Focusable - Dark Mode.pressed.chromium.png|489| Changed|
|Link Converged - Rendered as anchor.Stand-alone Disabled Focusable.pressed.chromium.png|873| Changed|

</details>


  <details><summary><b>Link Converged - Rendered as button</b> 12 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Link Converged - Rendered as button.Inline - RTL.pressed.chromium.png|172| Changed|
|Link Converged - Rendered as button.Inverted - Dark Mode.pressed.chromium.png|704| Changed|
|Link Converged - Rendered as button.Inline Disabled Focusable.pressed.chromium.png|623| Changed|
|Link Converged - Rendered as button.Inline.pressed.chromium.png|172| Changed|
|Link Converged - Rendered as button.Inverted - High Contrast.pressed.chromium.png|442| Changed|
|Link Converged - Rendered as button.Inverted disabled.pressed.chromium.png|516| Changed|
|Link Converged - Rendered as button.Inverted disabled - Dark Mode.pressed.chromium.png|909| Changed|
|Link Converged - Rendered as button.Inverted disabled - High Contrast.pressed.chromium.png|564| Changed|
|Link Converged - Rendered as button.Inverted.pressed.chromium.png|405| Changed|
|Link Converged - Rendered as button.Stand-alone Disabled Focusable.pressed.chromium.png|873| Changed|
|Link Converged - Rendered as button.Stand-alone - RTL.pressed.chromium.png|420| Changed|
|Link Converged - Rendered as button.Stand-alone.pressed.chromium.png|420| Changed|

</details>




<div id="vrtCommentfluent-ui_VRT_Pipeline_v9"/>